### ISSUE TITLE
Installation tfjs-node@3.21.0 doesn't make `dist` folder

### ISSUE THREAD

**Issue by tauraloke** [tensorflow/tfjs#6909]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):  Ubuntu 20.04.1 WSL; also Windows 10 itself.
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: none
- TensorFlow.js installed from (npm or script link): npm
- TensorFlow.js version: @tensorflow/tfjs-node@3.21.0
- CUDA/cuDNN version: none

**Describe the problem**
Installation doesn't make `dist` folder (without any error messages).
Downgrading to `@tensorflow/tfjs-node@3.20.0` is making `dist` folder correctly in the same environment.

**Provide the exact sequence of commands / steps that you executed before running into the problem**
```
mkdir test_bug-3-21-0 && cd test_bug-3-21-0
npm install --save @tensorflow/tfjs-node@3.21.0
stat ./node_modules/@tensorflow/tfjs-node/dist
```
Expected result: `bash: ./node_modules/@tensorflow/tfjs-node/dist: No such file or directory`

**Any other info / logs**

Log example without any problems:
```
$ npm rebuild @tensorflow/tfjs-node --verbose
npm verb cli /home/tauraloke/.nvm/versions/node/v16.16.0/bin/node /home/tauraloke/.nvm/versions/node/v16.16.0/bin/npm
npm info using npm@8.19.2
npm info using node@v16.16.0
npm timing npm:load:whichnode Completed in 0ms
npm timing config:load:defaults Completed in 1ms
npm timing config:load:file:/home/tauraloke/.nvm/versions/node/v16.16.0/lib/node_modules/npm/npmrc Completed in 1ms
npm timing config:load:builtin Completed in 1ms
npm timing config:load:cli Completed in 1ms
npm timing config:load:env Completed in 1ms
npm timing config:load:file:/home/tauraloke/projects/image_catalogizer/tensorflow-js-21/.npmrc Completed in 0ms
npm timing config:load:project Completed in 3ms
npm timing config:load:file:/home/tauraloke/.npmrc Completed in 0ms
npm timing config:load:user Completed in 0ms
npm timing config:load:file:/home/tauraloke/.nvm/versions/node/v16.16.0/etc/npmrc Completed in 0ms
npm timing config:load:global Completed in 1ms
npm timing config:load:validate Completed in 0ms
npm timing config:load:credentials Completed in 0ms
npm timing config:load:setEnvs Completed in 1ms
npm timing config:load Completed in 9ms
npm timing npm:load:configload Completed in 9ms
npm timing npm:load:mkdirpcache Completed in 2ms
npm timing npm:load:mkdirplogs Completed in 0ms
npm verb title npm rebuild @tensorflow/tfjs-node
npm verb argv "rebuild" "@tensorflow/tfjs-node" "--loglevel" "verbose"
npm timing npm:load:setTitle Completed in 1ms
npm timing config:load:flatten Completed in 2ms
npm timing npm:load:display Completed in 7ms
npm verb logfile logs-max:10 dir:/home/tauraloke/.npm/_logs
npm verb logfile /home/tauraloke/.npm/_logs/2022-10-06T06_50_44_061Z-debug-0.log
npm timing npm:load:logFile Completed in 4ms
npm timing npm:load:timers Completed in 0ms
npm timing npm:load:configScope Completed in 0ms
npm timing npm:load Completed in 25ms
npm timing arborist:ctor Completed in 0ms
npm timing arborist:ctor Completed in 0ms
npm timing build:queue Completed in 1ms
npm info run @tensorflow/tfjs-node@3.20.0 install node_modules/@tensorflow/tfjs-node node scripts/install.js
npm info run @tensorflow/tfjs-node@3.20.0 install { code: 0, signal: null }
npm timing build:run:install:node_modules/@tensorflow/tfjs-node Completed in 166ms
npm timing build:run:install Completed in 166ms
npm timing build:deps Completed in 169ms
npm timing build Completed in 170ms
rebuilt dependencies successfully
npm timing command:rebuild Completed in 251ms
npm verb exit 0
npm timing npm Completed in 319ms
npm info ok 
```

**Comment by datWeazel** [comment]
Similar issue for me. 
I got a node application using @tensorflow/tsjf-node@3.18.0 which suddenly stopped creating the `dist` directory during `npm install`. Worked yesterday without problems. Updating to @3.20.0 or @3.21.0 didn't work. Neither version creates the `dist` folder for me. 

Some more infos about my application: 
- Node application
- Running in docker (Container based on image node:lts)
- package.json got `"@tensorflow/tfjs": "^3.18.0",` and  `"@tensorflow/tfjs-node": "^3.18.0",` as regular dependencies
- `npm install` is run inside the container to install dependencies from package.json
- no `node_modules/@tensorflow/tfjs-node/dist` folder created at all

**Comment by mattsoulanille** [comment]
Sorry about this. There's likely a bug with the publishing script. I've published 3.21.1 with the `dist/` folder and deprecated @tensorflow/tfjs-node 3.21.0.

**Comment by jjeshta** [comment]
> Sorry about this. There's likely a bug with the publishing script. I've published 3.21.1 with the `dist/` folder and deprecated @tensorflow/tfjs-node 3.21.0.

Thank you!
